### DISCUSSION TITLE
No picture, only layout information

### DISCUSSION THREAD

**Discussion by Vithanco** [hpcc-systems/hpcc-js-wasm#86] [Ideas]
Hi There,

This looks great! Thanks a lot for turning this great library into a WASM. 
 
I would like to use this great library not to create a drawing but to receive the layout information back. So, I would like to get for each node the location after the layouting, for each edge the path and for each cluster the location. I know that the original library has such a capability (https://graphviz.org/pdf/libguide.pdf)

Is such a feature already on the todo list?

Cheers,
Klaus

**Comment by GordonSmith** [comment]
Sounds like you may want to pick a different output format (JSON?):  https://github.com/hpcc-systems/hpcc-js-wasm#layout

**Reply by Vithanco** [reply]
Gordon, you are a genius! Thank you!
